bricks/diagram/src/draw-canvas/LineConnectorComponent.tsx

```
@@ -53,15 +62,19 @@ export function LineConnectorComponent({
 
+  let source: Cell | undefined;
+  let target: Cell | undefined;
+
   const available =
     !disabled &&
     hoverState &&
     (!!smartConnectLineState ||
-      (lineEditorState
+      (activeEditableEdge &&
+      lineEditorState &&
+      (({ source, target } = editableLineMap.get(activeEditableEdge)!), true)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)